languages/python/exercises/concept/bools/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise, you need to translate some rules from the classic game Pac-Man into Elixir functions.
```

Comment:
I realize that Pac-Man is as famous as any game, but do we need to include a link to Wikipedia or something here?